vault/resource_pki_secret_backend_role.go

```diff
@@ -593,12 +601,12 @@ func pkiSecretBackendRoleRead(_ context.Context, d *schema.ResourceData, meta in
 		switch {
 		case k == consts.FieldNotBeforeDuration:
 			d.Set(k, flattenVaultDuration(secret.Data[k]))
-		case k == consts.FieldKeyBits:
-			keyBits, err := secret.Data[consts.FieldKeyBits].(json.Number).Int64()
+		case k == consts.FieldKeyBits || k == consts.FieldSignatureBits:
+			keyBits, err := secret.Data[k].(json.Number).Int64()
```

Comment:
signature_bits was added in Vault 1.10, I *think* this might bomb on Vault 1.9 instances, would you mind testing?

Reply:
Please ignore, we only support Vault 1.11.x and higher. See https://registry.terraform.io/providers/hashicorp/vault/latest/docs/guides/version_4_upgrade#what-vault-server-versions-are-supported-in-version-4x